### PR TITLE
Updated RESOURCES.md

### PR DESCRIPTION
**Pull request by yashgangwar7558**
Added a resource for K8ssandra and how we can connect Cassandra to Kubernetes using K8ssandra.

Addresses issue #25

### CODE REVIEW

File: RESOURCES.md [hunk 1]
```diff
@@ -25,6 +26,10 @@
     1. [Intro to Cassandra - DataStax](https://youtube.com/playlist?list=PL2g2h-wyI4SqCdxdiyi8enEyWvACcUa9R)
     2. [Learn Apache Cassandra - DataStax](https://youtube.com/playlist?list=PL2g2h-wyI4SqzmsHQZaYcK9uDmPuihPz2)
 
+- ### K8ssandra
```
Comment (kaiwalyakoparkar) [inline RESOURCES.md:29]: Please add this section after Kubernetes resources

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: RESOURCES.md (hunk 1 above)
Comment (patchandpray) [reply RESOURCES.md:29]: Agreed, @yashgangwar7558 to have this come after the kubernetes header is more logical :)

**Comment by yashgangwar7558** [comment]
@kaiwalyakoparkar @patchandpray I have made the suggested changes.

**Review by patchandpray — APPROVED** [review]

**Comment by patchandpray** [comment]
@kaiwalyakoparkar you requested changes, are the changes ok for you now? it is blocking merging :)

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**